Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -266,10 +268,12 @@ public void onShareLinkDialogDismissed() {
 
                             @Override
                             public void onLinkShareResponse(String sharedLink, String sharedChannel, BranchError error) {
+                                branchUniversalObject.userCompletedAction(BranchUniversalObject.BUO_USER_ACTIONS.SHARE_COMPLETED);
                             }
 
                             @Override
                             public void onChannelSelected(String channelName) {
+                                branchUniversalObject.userCompletedAction(BranchUniversalObject.BUO_USER_ACTIONS.SHARE_STARTED);
```

Comment:
Oh my bad. I thought so too but that's a good point. However, I think these are some of those events that we could add a ton of value to the dash without being distracting.